chainer/device_resident.py

```diff
@@ -56,28 +56,39 @@ def xp(self):
             return None
         return device.xp
 
-    def to_cpu(self):
-        # type: () -> 'DeviceResident'
+    def to_cpu(
+            self,
+            allow_unchain=None,  # type: tp.Optional[bool]
+    ):
+        # type: (...) -> 'DeviceResident'
         """Copies parameter variables and persistent values to CPU.
 
         This method does not handle non-registered attributes. If some of such
         attributes must be copied to CPU, the link implementation should
         override :meth:`~DeviceResident.device_resident_accept` to do so.
 
+        Args:
+            allow_unchain (bool): If ``True``, always unchains the graph. If
+                ``False``, stops and raises an error in case a graph would be
```

Comment:
* "stops and" can be removed (readers would wonder whether it has some special meaning).
* "in case a graph would be unchained.": How about the opposite logic e.g.  "in case the link was connected to a graph" or something? "would be unchained" has some ambiguity: readers might think it applies to the case where the variables were already unchained (especially those who are not very good at English).

Reply:
Fixed the docs.